package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-fh-build",
   "description": "A plugin for development and build lifecycle of FeedHenry components",
-  "version": "1.0.3",
+  "version": "2.0.0",
```

Comment:
Out of curiosity, what do we expect would happen if someone blindly upgraded to this version while still using nodejs@4 and npm@2. Should they expect to get the same bundles as they did before, or will there be some things different?